### ISSUE TITLE
Gps not working

### ISSUE THREAD

**Issue by Brajks** [MoKee/android_device_xiaomi_leo#6]
Hy all! 
Mokee is a lovely rom, sadly without gps working

**Comment by sarvesvara** [comment]
Amazing ROM with features. Thanks.
Same here.
GPS never worked in 6.0.1 version. Same with Ivan ROM - but that has some fix.

**Comment by ghost** [comment]
no, Gps is working on ivan rom, only locating using networks is not working and that's what the fix is correcting, on mokee location using networks is working but gps is not working. so it's the opposite.

**Comment by sarvesvara** [comment]
Thanks for the clarification @JeanPrendnote 
End of the day, turn-by-turn navigation is not working in the ROM.

**Comment by sarvesvara** [comment]
I had a bit of trouble over the weekend with the non-functional GPS, and ended up using the GPS in the iPhone.

So, I am back to MIUI ROM, until there is a GPS fix.

**Comment by sarvesvara** [comment]
@JeanPrendnote Is there a fix for this?

**Comment by ghost** [comment]
I don't know, i have throw this phone it was a real pain 🤐 the software suck, the SoC is heating like hell and battery life not the best :/
